src/main/java/htsjdk/variant/variantcontext/writer/SortingVariantContextWriterBase.java

```diff
@@ -132,6 +132,11 @@ public synchronized void add(VariantContext vc) {
         emitSafeRecords();
     }
 
+    @Override
+    public void setVcfHeader(VCFHeader header) {
+        innerWriter.setVcfHeader(header);
```

Comment:
but don't worry about this one since SortingVariantContextWriterBase is basically ununsed.